src/components/button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default class Button extends React.Component {
+  render() {
+
+    return (
+      <button onClick={this.props.resetSelection}>
+        Reset
+    </button>
+
+    );
+  }
+}
+
+Button.propTypes = {
+  resetSelection: React.PropTypes.isfunc,
```

Comment:
Close! I made a similar error. For a function proptype, the syntax is React.PropTypes.func , generally you'd also want to make it required which looks like this: React.PropTypes.func.isRequired